src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import data from '../material/bookdata.json'
```

Comment:
You could write
` import { books } from '../material/bookdata.json'`
And just write 
`books.map(...`

Reply:
Changed